specification/containerinstance/resource-manager/Microsoft.ContainerInstance/preview/2024-05-01-preview/containerInstance.json

```diff
@@ -2093,6 +2593,36 @@
         }
       }
     },
+    "ContainerGroupProfileListResult": {
+      "description": "The container group profile list response that contains the container group profile properties.",
+      "type": "object",
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ContainerGroupProfile"
+          },
+          "description": "The list of container group profiles."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URI to fetch the next page of container group profiles."
+        }
+      }
+    },
+    "ContainerGroupProfilePatch": {
+      "type": "object",
+      "description": "Properties of container group profile that need to be patched",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "description": "Resource tags.",
+          "additionalProperties": {
```

Comment:
>additionalProperties [](http://example.com/codeflow?start=11&length=20)

What is this field used for?

Reply:
Tags is basically a dictionary of where both the key and value would be strings, something like this - 
"tags": {
  "tag1key": "tag1Value",
  "tag2key": "tag2Value"
}
That is why I have initialized it using the additionalProperties parameter